test/tests/customhighlights.js

```diff
@@ -0,0 +1,77 @@
+"use strict";
+
+const expect = require("chai").expect;
+
+const Client = require("../../src/client");
+
+const client = new Client({}, "test", {clientSettings: {highlights: "foo, @all,   sp ace   , 고"}});
```

Comment:
i'd put it on a newline so it's very clear what you're testing against (maybe even `[..., ..., ...].join(',')`)